### PR TITLE
Move to GitHub actions

### PR DESCRIPTION
**Pull request by MancunianSam**
This only deploys to the management account so it needs its own custom deploy job.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        sbt assembly
+    secrets:
+      MANAGEMENT_ACCOUNT: ${{ secrets.MANAGEMENT_ACCOUNT }}
+      WORKFLOW_PAT: ${{ secrets.WORKFLOW_PAT }}
+  deploy:
+    needs: pre-deploy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: gh workflow run deploy.yml -f environment=intg -f to-deploy=${{ needs.pre-deploy.outputs.next-version }}
```
Comment (TomJKing) [inline .github/workflows/build.yml:22]: Should the `environment` parameter be set to `intg`?

**Review by TomJKing — COMMENTED** [review]
Looks good. One minor comment. Thanks

File: .github/workflows/build.yml (hunk 1 above)
Comment (MancunianSam) [reply .github/workflows/build.yml]: Good spot, the environment parameter shouldn't be there at all, the deploy job doesn't take on as this only gets deployed to management, I've taken it out.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**